lib/common/client/configuration/base.rb

```diff
@@ -92,23 +92,30 @@ def request_options
         def breakers_service
           return @service if defined?(@service)
 
+          @service = create_new_breakers_service(breakers_matcher, breakers_exception_handler)
```

Comment:
Rubocop was yelling at me about complexity so I split out exception_handler and matcher into methods.  the intended change is `elsif exception.is_a?(Faraday::ClientError)`

Reply:
Cool, I think this reads better anyways when broken up 👍